### ISSUE TITLE
Circus Parser requires non-existant file

### ISSUE THREAD

**Issue by thosvarley** [spyking-circus/spyking-circus#372]
I am following the example script given here: https://spyking-circus.readthedocs.io/en/latest/advanced/analysis.html

`from circus.shared.parser import CircusParser`
`from circus.shared.files import load_data`
`from pylab import *`
`params    = CircusParser('yourdatafile.dat')`
`results   = load_data(params, 'results')`
`spikes    = results['spiketimes']['temp_i']`
`isis      = numpy.diff(spikes)`
`hist(isis)`

Only one file was created with a *.dat suffix, but when I try and run:

`params    = CircusParser('data000000_all_sc.dat')`, it throws the following error:

`FileNotFoundError: [Errno 2] No such file or directory: '/media/spornslab/61cd4cb4-05b2-44d8-aa8d-9839488d41d7/DPT/01_02_03_processed/data000000_all_sc.params'`

**Comment by yger** [comment]
Indeed, because you should do CircusParser('data.dat'). Always target your main datafile, SC will automatically understand (if the overwrite flag was set to False) that it has to read one other file
